### DISCUSSION TITLE
Windows10的磁贴（大小、图标）

### DISCUSSION THREAD

**Discussion by ItIsNotAPlayer** [Meloong-Git/PCL#372] [提问] [answered]
一个没啥大用的功能：在PCL2固定在Windows10“开始”屏幕上时，能够将图标大小设为宽、大，且在中、宽、大时使用有PCL三个字母的图标。

**Comment by WForst-Breeze** [comment]
有新功能建议请前往 Issues 提交[新功能提案](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E6%96%B0%E5%8A%9F%E8%83%BD%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=feature.yml)

**Comment by Big-Cake-jpg** [comment, marked as answer]
传统 Win32（.exe） 应用在固定到开始菜单后只有“小”、“中”磁贴大小可以选择，其他大小的磁贴仅 UWP 可用。或者你可以尝试一下 edgeTile
